Answers.md

```diff
@@ -0,0 +1,50 @@
+1. What is the differnce between Responsive & Adaptive website?
+
+	- I would say the primary difference is that an adaptive site
+	will change a few key features, such as a navigation bar or
+	changing text from a row to a column.
+
+	- A fully responsive site completely adjusts to fit as many
+	screens as it possibly can while still looking neat. It's
+	a lot more work but it's very worth it. As how we have been
+	setting up every page makes them very easy to modify
+
+
+2. Mobile first VS Desktop first
+
+	- Building something 'X'-first means that your original page
+	or file you start with. Is being made for that platform.
+	So building something desktop first is starting with the page
+	being made for a desktop user. vis-a-vers-a
+
+
+3. What does 'font-size: 62.5%' in the 'html' tag do?
+
+	- Something awesome. With using REM units, it makes the text 
+	displaying on said page look extrememly similar cross platform.
+
+
+4. How would I describe preprocessing w/o terminology?
+
+	- "Less is used to make writing CSS easier. CSS just by itself can
+	get very overwhelming and extremely hard to organize when the project
+	starts getting really big. So Less is used to write cleaner looking
+	and reading code that is just easier to read, then it reformats the
+	Less code into CSS code. I like it a lot."
+
```

Comment:
Yup spot on! The one thing is its not just Less that does it. That's pretty much what ALL preprocessors do!